tony-history-server/app/hadoop/Requirements.java

```diff
@@ -19,38 +22,44 @@
 @Singleton
 public class Requirements {
   private static final Logger.ALogger LOG = Logger.of(Requirements.class);
+  private static Path histFolder;
   private static Path interm;
   private static Path finished;
 
+  private void createDirIfNotExists(FileSystem myFs, Path dir, FsPermission perm) {
+    String errorMsg;
+    try {
+      if (!myFs.exists(dir)) {
+        errorMsg = dir + " doesn't exist";
+        LOG.error(errorMsg);
```

Comment:
Let's issue a warn message instead of an error, since this could happen the first time THS is launched.

Reply:
👍